Formula/postgresql@11.rb

```diff
@@ -48,9 +48,6 @@ def install
     # The CLT is required to build Tcl support on 10.7 and 10.8 because
     # tclConfig.sh is not part of the SDK
     args << "--with-tcl"
-    if File.exist?("#{MacOS.sdk_path}/System/Library/Frameworks/Tcl.framework/tclConfig.sh")
```

Comment:
```suggestion
    if OS.mac? && File.exist?("#{MacOS.sdk_path}/System/Library/Frameworks/Tcl.framework/tclConfig.sh")
```

Don't delete those lines. Just check if we are on mac.

Reply:
@dawidd6 Ah, makes sense! What's the reason to have to check for `OS.mac?` if this is the `linuxbrew` repo? Do these ever get used on machines running macOS?

Reply:
It's because we can easily track the diff between `homebrew-core` (upstream) and `linuxbrew-core` (downstream). It allows us to do the merges easier.

Reply:
++ Makes sense!

Reply:
Amended the change!